docs/admin/multiple-schedulers.md

```diff
@@ -0,0 +1,235 @@
+      name: multischeduler-example
+  spec:
+    containers:
+    - name: pod-with-no-annotation-container
+      image: gcr.io/google_containers/pause:2.0
+  ```
+
+  When no scheduler annotation is supplied, the pod is automatically scheduled using the
+  default-scheduler.
+
+  Save this file as `pod1.yaml` and submit it to the Kubernetes cluster.
+
+  ```shell
+  $ kubectl create -f pod1.yaml
+  ```
+2. Pod spec with `default-scheduler` annotation
+
+  ```yaml
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: annotation-default-scheduler
+    annotations:
+      scheduler.alpha.kubernetes.io/name: default-scheduler
+    labels:
+      name: multischeduler-example
+  spec:
+    containers:
+    - name: pod-with-default-annotation-container
+      image: gcr.io/google_containers/pause:2.0
+  ```
+
+  A scheduler is specified by supplying the scheduler name as a value to the annotation
+  with key `scheduler.alpha.kubernetes.io/name`. In this case, we supply the name of the
+  default scheduler which is `default-scheduler`.
+
+  Save this file as `pod2.yaml` and submit it to the Kubernetes cluster.
+
+  ```shell
+  $ kubectl create -f pod2.yaml
+  ```
+3. Pod spec with `my-scheduler` annotation
+
+  ```yaml
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: annotation-second-scheduler
+    annotations:
+      scheduler.alpha.kubernetes.io/name: my-scheduler
+    labels:
+      name: multischeduler-example
+  spec:
+    containers:
+    - name: pod-with-second-annotation-container
+      image: gcr.io/google_containers/pause:2.0
+  ```
+
+  In this case, we specify that this pod should be scheduled using the scheduler that we
+  deployed - `my-scheduler`. Note that the value of the annotation with key
+  `scheduler.alpha.kubernetes.io/name` should match the name supplied to the scheduler
+  command as an argument in the deployment config for the scheduler.
+
+  Save this file as `pod3.yaml` and submit it to the Kubernetes cluster.
+
+  ```shell
+  $ kubectl create -f pod3.yaml
+  ```
+
+  Verify that all three pods are running.
+
+  ```shell
+  $ kubectl get pods
+  ```
+
+### Verifying that the pods were scheduled using the desired schedulers
+
+In order to make it easier to work through these examples, we did not verify that the
+pods were actually scheduled using the desired schedulers. We can verify that by
+changing the order of pod and deployment config submissions above. If we submit all the
+pod configs to a Kubernetes cluster before submitting the scheduler deployment config,
+we see that the pod `annotation-second-scheduler` remains in "Pending" state forever
+while the other two pods get scheduled. Once we submit the scheduler deployment config
+and our new scheduler starts running, the `annotation-second-scheduler` pod gets
+scheduled as well.
+
+Alternatively, one could just look at the "Scheduled" entries in the event logs to
+verify that the pods were scheduled by the desired schedulers.
+
+```shell
+$ kubectl get events
```

Comment:
Because it complicates copy/paste for people. Pretty soon all code examples are going to have click-to-copy. The "$" will break it. Not only that, but even if we wanted to show a dollar sign to signify the shell prompt we could do it with a CSS injection. And not only all that, it is standard procedure for all Google cloud docsets and an established Markdown standard. 

http://www.cirosantilli.com/markdown-style-guide/#dollar-signs-in-shell-code